packages/nimble-components/src/rich-text-mention/mention-users/index.ts

```diff
@@ -0,0 +1,77 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { MentionInternalsOptions } from '../base/models/mention-internals';
+import type { MappingMentionBase } from '../../mapping/mention-base';
+import { RichTextMentionUsersValidator } from './models/rich-text-mention-users-validator';
+import type { RichTextMentionValidity } from '../base/models/mention-validator';
+import {
+    MappingConfigs,
+    RichTextMentionBase,
+    RichTextMentionConfig
+} from '../mention-base';
+import type { MappingConfig } from '../mention-base/models/mapping-config';
+import { MappingUserConfig } from '../mention-base/models/mapping-user-config';
+import { MappingMentionUser } from '../../mapping/mention-user';
+import { template } from '../mention-base/template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-rich-text-mention-users': RichtextMentionUsers;
+    }
+}
+
+/**
+ * Rich Text Mention that will map user url and name
+ */
+export class RichtextMentionUsers extends RichTextMentionBase<
+RichTextMentionConfig,
+RichTextMentionUsersValidator
+> {
+    private readonly character = '@';
+
+    private readonly icon = '';
+
+    public override createValidator(): RichTextMentionUsersValidator {
+        return new RichTextMentionUsersValidator(this.mentionInternals);
```

Comment:
```suggestion
        return new RichTextMentionBaseValidator(this.mentionInternals, baseValidityFlagNames, [MappingMentionUser]);
```
I don't think you need a separate `RichTextMentionUsersValidator` class, since it doesn't add anything to the base validator. Just remove `abstract` from the base validator class so you can construct it. (Then remove the file for the other validator.)

Reply:
Updated